pkg/mkimage-raw-efi/install

```diff
@@ -156,7 +156,7 @@ if [ -n "$CONFIG_PART" ] ; then
    ln -s /dev/$CONFIG_PART /parts/config.img
 fi
 
-for i in persist.img config.img rootfs.img EFI ; do
+for i in persist.img config.img rootfs.img EFI boot; do
```

Comment:
So looking at this got me thinking that this could be our chance to potentially get rid of `boot` hack altogether. IIRC, the last reason we need it (as opposed to simply using `efi` and `do_efi` is the need for `trampoline.grub.cfg` on PRi4. Can you please check if we still need this @giggsoff and if we don't lets go ahead and get rid of separate `boot` handling, ok?

Reply:
To synchronize the actions performed in do_efi and do_boot, I tried:

```
diff --git a/pkg/mkimage-raw-efi/make-raw b/pkg/mkimage-raw-efi/make-raw
--- a/pkg/mkimage-raw-efi/make-raw	(revision e3ff60d9d2c35cb03042e90ce2578984fb7e073e)
+++ b/pkg/mkimage-raw-efi/make-raw	(revision cda511c2f3c22af22057f9b4155dcf4ea7053fc5)
@@ -141,7 +141,7 @@
 
 do_boot() {
   mkefifs
-  cp /trampoline.grub.cfg /efifs/EFI/BOOT/grub.cfg
+  sed -e 's#@PATH_TO_GRUB@#'"$(cd /efifs; echo EFI/BOOT/BOOT*EFI)"'#' < /grub.cfg.in > /efifs/EFI/BOOT/grub.cfg
   cp -r /parts/boot/* /efifs/
   do_system_vfat_part "$1" "$BOOT_PART_SIZE"
 }
```

And it works fine on my RPi. So, seems, we can merge do_efi and do_boot. Shall I do it in this PR?

Reply:
Please send a separate PR -- once that other PR gets merged you can rebase this PR on top of it @giggsoff